etl/tests/integration/pipeline_v2_test.rs

```diff
@@ -542,6 +542,34 @@ async fn test_table_copy() {
     let age_sum =
         get_users_age_sum_from_rows(&destination, database_schema.users_schema().id).await;
     assert_eq!(age_sum, expected_age_sum);
+
+    // Check that the replication slots for the two tables have been removed.
+    let users_replication_slot = get_slot_name(
```

Comment:
Instead of creating new tests, I decided to just assert the slots at the end of existing tests.